tests/backwards_compatibility_tests/base_test_case.py

```diff
@@ -0,0 +1,58 @@
+import json
+import logging
+import os
+import sys
+from pathlib import Path
+
+from marqo_test import MarqoTestCase
+
+sys.path.append(os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__)))))
+
+class BaseCompatibilityTestCase(MarqoTestCase):
+    """
+    Base class for backwards compatibility tests. Contains a prepare method that should be implemented by subclasses to
+    add documents / prepare marqo state. Also contains methods to save and load results to/from a file so that
+    test results can be compared across versions.
+    """
+    @classmethod
+    def setUpClass(cls) -> None:
+        super().setUpClass()
+        cls.logger = logging.getLogger(cls.__name__)
+        handler = logging.StreamHandler()
+        formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+        handler.setFormatter(formatter)
+        cls.logger.addHandler(handler)
+        cls.logger.setLevel(logging.INFO)
```

Comment:
is this configurable somehow? we log debug info in class methods.

Reply:
added a method to make it configurable: https://github.com/marqo-ai/marqo/pull/1024/commits/740097f76bc5fd67568d409c1a680d9ebcb77333#diff-3b756b38875ef9e366500400a91b87df9b4a91e7a547e2de0a1886fcc605ca50R61-R68